src/schema/v2/sale_artwork.ts

```diff
@@ -145,6 +146,32 @@ export const SaleArtworkType = new GraphQLObjectType<any, ResolverContext>({
         description: "Singular estimate field, if specified",
         resolve: ({ estimate_cents }) => estimate_cents,
       },
+      formattedStartDateTime: {
+        type: GraphQLString,
+        description:
+          "A formatted description of when the lot starts or ends or if it has ended",
+        resolve: (saleArtwork, _options, { defaultTimezone, saleLoader }) => {
+          if (!!saleArtwork.sale) {
```

Comment:
I was sort of pattern matching here based on whether or not `sale` is present on `saleArtwork` and then to use `sale_id` with a loader if it's not. It seems messy to me. Is it OK to just assume `sale_id` is present?

Reply:
Yea, I think you can assume `sale` isn't present and fetch the sale. It looks like `sale` isn't included in sale artwork JSON so depending on how this is fetched, `sale` wouldn't be present.

Reply:
Cool that's simpler